core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.spec.ts

```diff
@@ -0,0 +1,175 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+/**
+ * @fileoverview Unit tests for the skills list directive.
+ */
+
+import { UpgradedServices } from 'services/UpgradedServices';
+
+describe('Skills List Directive', function() {
+  beforeEach(angular.mock.module('oppia'));
+
+  beforeEach(angular.mock.module('oppia', function($provide) {
+    var ugs = new UpgradedServices();
+    for (let [key, value] of Object.entries(ugs.getUpgradedServices())) {
+      $provide.value(key, value);
+    }
+  }));
+  var $uibModal = null;
+  var $scope = null;
+  var ctrl = null;
+  var $q = null;
+  var $httpBackend = null;
+  var $rootScope = null;
+  var directive = null;
+  var $timeout = null;
+  var EditableTopicBackendApiService = null;
+  var TopicsAndSkillsDashboardBackendApiService = null;
+
+  beforeEach(angular.mock.inject(function($injector) {
+    $uibModal = $injector.get('$uibModal');
+    $rootScope = $injector.get('$rootScope');
+    $scope = $rootScope.$new();
+    $httpBackend = $injector.get('$httpBackend');
+    $timeout = $injector.get('$timeout');
+    $q = $injector.get('$q');
+    TopicsAndSkillsDashboardBackendApiService =
```

Comment:
Wrap inside (..) and reak after (. Here and elsewhere in this file.

Reply:
Done.